topics/proteomics/tutorials/proteome_annotation/tutorial.md

```diff
@@ -0,0 +1,323 @@
+---
+layout: tutorial_hands_on
+
+title: Annotating a protein list identified by LC-MS/MS experiments
+
+zenodo_link: 
```

Comment:
We currently can deal with only one Zenodo link. Could you create one Zenodo with all the data you need there?

Reply:
In the description of the Zenodo entry, you can link to the other Zenodo entries

Reply:
Done.

Reply:
It should be only one Zenodo link there. Do you have one Zenodo with all data inside?

Reply:
in the zenodo description of this dataset : 
https://zenodo.org/record/2650868
I referenced the 2 other datasets. 
Isn't it what you asked me to do ? 
If there's is a more formal way to reference datsets, I didn't find it

Reply:
We need one Zenodo repository with all 3 datasets in it (not just references). Could you update the existing Zenodo repository and add the other two files (and give me the new link)?

Reply:
it is not possible to update an existing zenodo registered page by adding files. 
I thus created a new zenodo entry with the 3 files :
https://zenodo.org/record/3405119
please tell me if it is ok like this and if I should do something else ?